### PR TITLE
Cut v0.16.0 beta.0

### PR DESCRIPTION
**Pull request by mxinden**
Cherry-picking pretty much everything from master over to release-0.16 for the next beta release.

Between beta and stable release I would only cherry-pick bug fixes. Preferably those would even go into the release-0.16 branch right away, merging them back into master afterwards. What do you think?

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,15 @@
+## 0.16.0-beta.0 / 2019-01-14
+
+* [ENHANCEMENT] Update prometheus/common and add support for --log.format (#1658)
+* [ENHANCEMENT] Add CORS support to API v2 (#1667)
+* [ENHANCEMENT] Support HTML, URL title and custom sounds for Pushover (#1634)
+* [ENHANCEMENT] [amtool] Support for assuming first label is alertname in silence add and query (#1693)
```
Comment (stuartnelson3) [inline CHANGELOG.md]: I would say this is a bugfix, since I believe it was already listed as possible in the documentation

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,15 @@
+## 0.16.0-beta.0 / 2019-01-14
+
+* [ENHANCEMENT] Update prometheus/common and add support for --log.format (#1658)
```
Comment (simonpasquier) [inline CHANGELOG.md]: I'd only keep the second part: `add support for --log.format`

**Comment by mxinden** [comment]
Thanks @stuartnelson3 @simonpasquier, updated accordingly.

The DCO bot is complaining about changed e-mail addresses. Given that all of these commits already made it into master, and given that they are all signed, I would suggest merging this PR. Any objections?

(I will run this through the Prometheus Operator test suit as a smoke test first).

**Comment by stuartnelson3** [comment]
sounds good to me!

**Comment by mxinden** [comment]
I have included https://github.com/prometheus/alertmanager/pull/1656 as well.

The current state passes the Prometheus Operator tests as well: https://github.com/coreos/prometheus-operator/pull/2145

I will cut the release later tonight or tomorrow.

**Pull request merged**